app/templates/components/sl-split-grid.hbs

```diff
@@ -0,0 +1,111 @@
+<header class="grid-header">
+    <div class="row">
+        <div class="col-sm-6">
+            {{#if view.headerTemplate}}
+                <h3 class="title">
+                    {{partial view.headerTemplate}}
+                </h3>
+            {{/if}}
+        </div>
+
+        {{#if filterName}}
+            <div class="col-sm-6 text-right">
+                {{#sl-button action="toggleFilterPane" size="small"}}{{filterText}}{{/sl-button}}
+            </div>
+        {{/if}}
+    </div>
+</header>
+
+<div class="filter-pane">
+    {{#if filterOpen}}
+        <div class="filter-content">
+            {{split-grid-filter}}
+        </div>
+    {{/if}}
+</div>
+
+<div class="list-pane">
+    <table class="table column-headers">
+        <thead>
+            <tr>
+                {{#each column in columns itemViewClass="sl-split-grid-column-header"}}
+                    {{column.title}}
+                    {{#if column.sortable}}
+                        <i {{bind-attr class=":fa view.sortIconClass"}}></i>
+                    {{/if}}
+                {{/each}}
+
+                {{#if view.rowActions}}
+                    <th {{bind-attr style=actionsColumnStyle}} class="actions-cell"></th>
+                {{/if}}
+            </tr>
+        </thead>
+    </table>
+
+    <div class="content">
+        <table class="table table-hover table-striped">
+            <tbody>
+                {{#if view.rowActions}}
+                    {{#each row in content itemController=view.rowController itemViewClass="sl-split-grid-row"}}
+                        {{#each column in columns itemViewClass="sl-split-grid-cell"}}
+                            {{split-grid-cell}}
+                        {{/each}}
+
+                        <td {{bind-attr style=actionsColumnStyle}} class="actions-cell">
+                            {{#sl-drop-button align="right" label=actionsButtonLabel size="small" theme="hover"}}
+                                {{#each rowAction in rowActions}}
+                                    {{sl-drop-option action=rowAction.action label=rowAction.label}}
+                                {{/each}}
+                            {{/sl-drop-button}}
+                        </td>
+                    {{/each}}
+                {{else}}
+                    {{#each row in content itemController=view.rowController itemViewClass="sl-split-grid-row"}}
+                        {{#each column in columns itemViewClass="sl-split-grid-cell"}}
+                            {{split-grid-cell}}
+                        {{/each}}
+                    {{/each}}
+                {{/if}}
```

Comment:
I'm assuming a majority of this code is duplicated in this if/else check for performance gains?  If so, please add a (handlebars) comment stating such so that a less-informed developer will not view this as a refactoring opportunity.